client/factory.go

```diff
@@ -227,7 +227,9 @@ func (b *clientFactory) createGRPCConnection(c *cli.Context) (*grpc.ClientConn,
 
 func (b *clientFactory) createTLSConfig(c *cli.Context) (*tls.Config, error) {
 	certPath := c.String(common.FlagTLSCertPath)
+	certData := c.String(common.FlagTLSCertData)
 	keyPath := c.String(common.FlagTLSKeyPath)
+	keyData := c.String(common.FlagTLSKeyData)
```

Comment:
Thanks for sending this over! Would you mind adding a check to make sure the user only uses `-file` or `-data`, but doesn't mix and match (or use both)?

Reply:
Hi @bwebster I'm also interested in getting this one merged. Let us know if you need a hand!

Reply:
@taonic Sorry for the slow response.  I've had zero free time as of late.  If you want to push this forward, please feel free to.  Otherwise I can try and get back to it next week.